CHANGES.md

```diff
@@ -24,6 +24,16 @@ OpenSSL 3.2
 
 ### Changes between 3.0 and 3.2 [xx XXX xxxx]
 
+ * Removed all references to invalid OSSL_PKEY_PARAM_RSA names for CRT parameters
+   in OpenSSL code.
+   Applications should not use the names OSSL_PKEY_PARAM_RSA_FACTOR,
+   OSSL_PKEY_PARAM_RSA_EXPONENT and OSSL_PKEY_PARAM_RSA_COEFFICIENT.
+   Use the numbered names such as OSSL_PKEY_PARAM_RSA_FACTOR1 instead.
+   Using these invalid names may cause algorithms to use slower methods
+   that ignore the CRT parameters.
```

Comment:
Shouldn't something a bit stronger be here?

It will ignore everything you give it.  Multiprime won't be. ...

Reply:
Hmm, also the changes entry should be in 3.0 section as this is certainly going there too. We should aim for having it in 3.0.7.